modules/latex/latex.js

```diff
@@ -0,0 +1,8 @@
+( function( $ ) {
+	$( document.body ).on( 'post-load', function() {
+		// New posts have been added to the page. Re-typeset MathJax if necessary.
+		if ( typeof MathJax !== 'undefined' ) {
+			MathJax.Hub.Queue( [ 'Typeset', MathJax.Hub ] );
```

Comment:
This currently re-typesets the entire page. I'm working on a change that'll only typeset Math in the newly rendered content.